tests/test_queue.py

```diff
@@ -108,3 +162,46 @@ def test_transaction(var: str) -> str:
     with SetWorkflowID(wfid):
         assert test_transaction("abc") == "abc1"
     assert step_counter == 1
+
+
+def test_limiter(dbos: DBOS) -> None:
+
+    @DBOS.workflow()
+    def test_workflow(var1: str, var2: str) -> float:
+        assert var1 == "abc" and var2 == "123"
+        return time.time()
+
+    max = 5
+    duration = 2
+    queue = Queue("test_queue", limiter={"max": max, "duration": duration})
```

Comment:
I feel the name `max` and `duration` could be more specific. I know they're used by BullMQ, but it'd be clearer if you call them `max_outstanding` and `duration_second`, something like that.

Reply:
As discussed, change max -> limit. and duration -> period